android/app/src/main/java/covidsafepaths/bt/exposurenotifications/ExposureNotificationsModule.java

```diff
@@ -105,6 +106,11 @@ public void getCurrentENPermissionsStatus(final Callback callback) {
                         });
     }
 
+    @ReactMethod
```

Comment:
Should this go in the `DeviceInfo` module instead?

Reply:
Done!